.github/workflows/codeql-analysis.yml

```
@@ -51,8 +60,8 @@ jobs:
             # Vellum assets are compiled from another repo that's scanned separately
             - 'corehq/apps/app_manager/static/app_manager/js/vellum/'
             # Couch views are a special case
-            - '**/_design/**.js'
-            - 'corehq/couchapps/**.js'
+            - '**/_design/*.js'
+            - 'corehq/couchapps/*.js'
```

Seems like these will not match files in nested directories. Examples:

```
corehq/apps/users/_design/views/by_domain/map.js

corehq/couchapps/last_modified/views/by_last_modified/map.js
```

Does that not matter?
```suggestion
            - '**/_design/**/*.js'
            - 'corehq/couchapps/**/*.js'
```

---

oh dang you're right - that's probably why I did it that way in the first place.  Thank you.